examples/hellogrpc/e2e-test.sh

```diff
@@ -32,18 +33,23 @@ get_lb_ip() {
 apply-lb() {
   echo Applying service...
   bazel build examples/hellogrpc:staging-service.apply
-  bazel-bin/examples/hellogrpc/staging-service.apply
+  bazel run examples/hellogrpc/staging-service.apply
```

Comment:
I think using bazel-bin in one instance was intentional to ensure this method of deploying works correctly.